config/opal_check_os_flavors.m4

```diff
@@ -22,7 +22,7 @@ AC_DEFUN([OPAL_CHECK_OS_FLAVOR_SPECIFIC],
     AC_MSG_CHECKING([$1])
     AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
      [[#ifndef $1
-      error: this isnt $1
+      error: this isn't $1
```

Comment:
This should be `isnt` or `is not`, but not `isn't`.  The ' will cause all kinds of escaping fun in that file.

Reply:
reverted this fix